src/meltano/core/project_plugins_service.py

```diff
@@ -177,7 +177,7 @@ def remove_from_file(self, plugin: ProjectPlugin):  # noqa: ANN201
             The removed plugin.
         """
         # Will raise if the plugin isn't actually in the file
-        self.get_plugin(plugin)
+        self.get_plugin(plugin, ensure_parent=False)
```

Comment:
We were failing to remove the plugin from `meltano.yml` is the lockfile isn't present.